src/cclib/parser/data.py

```diff
@@ -131,7 +132,8 @@ class ccData(object):
        "nooccnos":         Attribute(numpy.ndarray,    'TBD',                         'N/A'),
        "optdone":          Attribute(bool,             'done',                        'optimization'),
        "optstatus":        Attribute(numpy.ndarray,    'status',                      'optimization'),
-       "polarizabilities": Attribute(list,             'polarizabilities',            'N/A'),
+       "polarizabilities": Attribute(list,             'polarizabilities',            'N/A'),

```

Comment:
This line shouldn't be marked. I didn't change this line, just omit this change.